docs/components-overview.md

```diff
@@ -48,42 +48,44 @@ The following table lists the .NET Aspire components currently available for use
 
-    ```json
-    {
-      "ConnectionStrings": {
-        "PostgreSqlConnection": "Host=myserver;Database=test"
-      }
-    }
-    ```
+1. In your orchestrator project (the project with the _*.AppHost_ suffix), add a reference to the Worker Service project. If you're using Visual Studio, you can use the [**Add .NET Aspire Orchestrator Support**](setup-tooling.md#add-orchestration-projects) project context menu item to add the reference automatically. The following code snippet shows the project reference of the _AspireApp.AppHost.csproj_:
 
-1. Inject the `NpgsqlDataSource` object into your controllers or service endpoints to run commands against the database:
+    :::code language="xml" source="snippets/components/AspireApp/AspireApp.AppHost/AspireApp.AppHost.csproj" highlight="16":::
 
-    ```csharp
-    public class ExampleService(NpgsqlDataSource dataSource)
-    {
-    }
-    ```
+    After the Worker Service project is referenced in the orchestrator project, the Worker Service project has its _Program.cs_ file updated to call the `AddServiceDefaults`. For more information on service defaults, see [Service defaults](service-defaults.md).
+
+1. In the orchestrator project, update the _Program.cs_ file with the following code:
+
+    :::code source="snippets/components/AspireApp/AspireApp.AppHost/Program.cs" highlight="3-4,6-8":::
+
+    The preceding code:
+
+    - Calls <xref:Aspire.Hosting.PostgresBuilderExtensions.AddPostgresContainer%2A> and chains a call to <xref:Aspire.Hosting.PostgresBuilderExtensions.AddDatabase%2A>, adding a PostgreSQL database container to the app model with a database named `"customers"`.
+    - Chains calls on the result of the <xref:Aspire.Hosting.ProjectResourceBuilderExtensions.AddProject%2A> from the Worker Service project:
+      - Calls <xref:Aspire.Hosting.ResourceBuilderExtensions.WithReference%2A> to add a reference to the `database`.
+      - Calls <xref:Aspire.Hosting.ProjectResourceBuilderExtensions.WithReplicas%2A> to set the number of replicas to `3`.
+
+1. Inject the `NpgsqlDataSource` object into the `Worker` to run commands against the database:
+
+    :::code source="snippets/components/AspireApp/WorkerService/Worker.cs" highlight="7,13":::
 
-After four steps, you now have a fully configured PostgreSQL database component integrated into your app. This component also configured health checks, logging, tracing, metrics, retries, and other useful capabilities for you behind the scenes. .NET Aspire components provide various options to configure each of these features.
+After several steps, you now have a fully configured PostgreSQL database component and corresponding container with connection integrated into your app. This component also configured health checks, logging, tracing, metrics, retries, and other useful capabilities for you behind the scenes. .NET Aspire components provide various options to configure each of these features.
```

Comment:
nit: would it be better to remove that intro? We assume they've done the steps, so we don't really have to call that out. Leave it in if you feel it's right. I know the way I talk and communicate comes out in the articles I write so if this (or anything I suggest) is more in line with how you're communicating the info, skip my suggestion. 😁 I added `!` for flair and excitement!

Additionally, you can cut out tracing to lower the amount of items listed. I would assume that if you can log you can trace? Maybe I'm wrong.

```suggestion
You now have a fully configured PostgreSQL database component and corresponding container with connection integrated into your app! This component also configured health checks, logging, metrics, retries, and other useful capabilities for you behind the scenes. .NET Aspire components provide various options to configure each of these features.
```